seed/views/v3/properties.py

```diff
@@ -45,3 +208,767 @@ def labels(self, request):
         super_organization = self.get_organization(request)
         # TODO: refactor to avoid passing request here
         return get_labels(request, labels, super_organization, 'property_view')
+
+    @swagger_auto_schema(
+        manual_parameters=[
+            AutoSchemaHelper.query_org_id_field(),
+            AutoSchemaHelper.query_integer_field(
+                'cycle',
+                required=False,
+                description='The ID of the cycle to get properties'),
+        ]
+    )
+    @api_endpoint_class
+    @ajax_request_class
+    @has_perm_class('requires_viewer')
+    def list(self, request):
+        """
+        List all the properties	with all columns
+        """
+        return self._get_filtered_results(request, profile_id=-1)
+
+    @swagger_auto_schema(
+        manual_parameters=[
+            AutoSchemaHelper.query_org_id_field(),
+            AutoSchemaHelper.query_integer_field(
+                'profile_id',
+                required=False,
+                description='Either an id of a list settings profile, or '
+                            'undefined'),
+            AutoSchemaHelper.query_array(
+                'cycle_ids',
+                item_type='integer',
+                required=True,
+                description='The IDs of the cycle to get properties',
+            )
+        ]
+    )
+    @api_endpoint_class
+    @ajax_request_class
+    @has_perm_class('requires_viewer')
+    @action(detail=False, methods=['GET'])
+    def filter_by_cycle(self, request):
+        """
+        List all the properties	with all columns
+        """
+        org_id = request.query_params.get('organization_id', None)
+        profile_id = request.query_params.get('profile_id', -1)
+        cycle_ids = request.query_params.get('cycle_ids', [])
```

Comment:
This is broken, drf-yasg gives query array as abc=1,2,3 while django expects abc=1&abc=2&abc=3. On my list to come back and fix this by just splitting on commas

Reply:
more context: https://github.com/axnsan12/drf-yasg/issues/317#issuecomment-486104837

Reply:
After a discussion, we decided to change the view http method to POST. Reasoning is that it's messy to split on commas (swagger method), we want to be able to do requests from swagger (django method prevents) and its overcomplicated to allow both.